frontend/src/common/ReviewWithComment.tsx

```diff
@@ -1,18 +1,20 @@
 import { LoadingButton } from '@mui/lab'
-import { Autocomplete, Stack, TextField, Typography } from '@mui/material'
+import { Autocomplete, Divider, Stack, TextField, Typography } from '@mui/material'
 import { useTheme } from '@mui/material/styles'
-import { SyntheticEvent, useMemo, useState } from 'react'
+import { SyntheticEvent, useEffect, useMemo, useState } from 'react'
+import { latestReviewsForEachUser } from 'utils/reviewUtils'
 
 import { useGetModelRoles } from '../../actions/model'
 import { useGetReviewRequestsForModel } from '../../actions/review'
-import { AccessRequestInterface, ReleaseInterface, ReviewRequestInterface } from '../../types/types'
+import { AccessRequestInterface, Decision, ReleaseInterface, ReviewRequestInterface } from '../../types/types'
 import { getRoleDisplay } from '../../utils/roles'
 import MessageAlert from '../MessageAlert'
 import Loading from './Loading'
 
 export const ResponseTypes = {
```

Comment:
I've noticed we have this defined in two places in the frontend. Can we remove this `ResponseTypes` and use `Decision` from `frontend/types/types.ts`?